src/number/format-properties.js

```diff
@@ -71,11 +71,11 @@ return function( pattern, cldr, options ) {
 		// the value to be formatted, though we're always using 1 as a simplification, because the
 		// number won't be zero-padded since we chose the right format based on the scale, i.e.,
 		// we'd never see something like `003M` anyway.
-		properties[ 2 ] = negativeSuffix[ 2 ] = 1; // minimumIntegerDigits
-		properties[ 3 ] = negativeSuffix[ 3 ] = 0; // minimumFractionDigits
-		properties[ 4 ] = negativeSuffix[ 4 ] = 0; // maximumFractionDigits
-		properties[ 5 ] = negativeSuffix[ 5 ] = // minimumSignificantDigits &
-			properties[ 6 ] = negativeSuffix[ 6 ] = undefined ; // maximumSignificantDigits
+		properties[ 2 ] = 1; // minimumIntegerDigits
+		properties[ 3 ] = 0; // minimumFractionDigits
+		properties[ 4 ] = 0; // maximumFractionDigits
+		properties[ 5 ] = // minimumSignificantDigits &
+			properties[ 6 ] = undefined ; // maximumSignificantDigits
```

Comment:
Please, what about setting the `negativeSuffix` items?

Reply:
Please, can you explain how this change fixes the problem? Thanks

Reply:
negativeSuffix is a string.

Setting `negativeSuffix[idx] = <value>` is not allowed in `strict mode` of javascript. You can try and reproduce it locally with:
```
(function strictMode() {
`use strict`;
let str = "";
str[2] = 1;
})()
```

Reply:
> Please, what about setting the `negativeSuffix` items?

I could not find a reason to set these to negativeSuffix in the first place. It will be great if you have some pointers towards that, so that I might come up with a workaround, if needed.

Reply:
That's true!! What I found out: I believe `negativeSuffix` was a typo. It should have been `negativeProperties`. Although, it also doesn't make sense to set `negativeProperties` at all. This variable is only used as an auxiliary variable to get `negativePrefix` and `negativeSuffix`. Therefore, I believe that's was mistake when compact support was implemented. That mistake doesn't cause bug, because this is variable isn't used and it's ignored.

Your change looks good to me.